services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -35,6 +35,8 @@ export const DropdownField = ({
   validateOnRender,
   sxOverride,
   styleAsOptional,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Fixes `Warning: Received 'true' for a non-boolean attribute 'clear'.` but isn't used anywhere in the component, so it causes a lint error.